shared/src/main/scala/mlscript/TyperHelpers.scala

```diff
@@ -1110,14 +1110,15 @@ abstract class TyperHelpers { Typer: Typer =>
             assert(td.tparams.size === targs.size)
             subst(td.body, td.tparams.lazyZip(targs).map {
               case (tp, w: WildcardArg) => 
-                // TODO raise type error here
-                SkolemTag(tp._2)(noProv) -> errType
+                // * FIXME we should NOT use TypeBounds here (somehow causes infinite subtyping loop)
+                // *       but raising an error will break a lot of  code related to alias (maybe changling `subst`)
```

Comment:
This should be an error. We must make sure type aliases never get wildcard args.

Reply:
it seems that wildcards can get into type aliases via extrusion currently

Reply:
Well extrusion must be changed so this doesn't happen.

Reply:
done